### PR TITLE
Add Kali template config

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

**Review by marmarek — APPROVED** [review]

File: R4.1/templates-community/kali.conf [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+GIT_BASEURL ?= https://github.com
+GIT_PREFIX ?= QubesOS/qubes-
+BRANCH ?= master
+
+RELEASE ?= 4.1
+
+VERBOSE = 2
+BACKEND_VMM=xen
+
+DIST_DOM0 ?= fc25
```
Comment (SaswatPadhi) [inline R4.1/templates-community/kali.conf:12]: May be this is not an issue, but I noticed that all the other R4.1 templates have `fc29` for `DIST_DOM0` ...

Comment (marmarek) [reply R4.1/templates-community/kali.conf:12]: In release builds this isn't an issue, as this config is used only for root.img part (where `DISTS_VM` matters), not packaging it into rpm (that would be done in `DIST_DOM0` environment). But lets make it consistent.
Thanks for noticing!

Comment (fepitre) [reply R4.1/templates-community/kali.conf:12]: Yes indeed, it's a mistake. A copy paste from R4.0 not totally reviewed :)

Comment (SaswatPadhi) [reply R4.1/templates-community/kali.conf:12]: No worries :) Thanks for all the great work you've been doing on Qubes.

**Pull request merged**